src/univariate/discrete/binomial.jl

```diff
@@ -105,6 +105,17 @@ function entropy(d::Binomial; approx::Bool=false)
     end
 end
 
+function kldivergence(p::Binomial, q::Binomial; kwargs...)
+    if ntrials(p) == ntrials(q)
+        return ntrials(p) * kldivergence(Bernoulli(succprob(p)), Bernoulli(succprob(q)))
+    elseif ntrials(p) > ntrials(q)
+        return Inf
```

Comment:
This is not typestable (sometimes, eg if parameters are of type Float32). The easiest and most reliable approach (that is commonly used in Distributions) is something like

```suggestion
    np = ntrials(p)
    nq = ntrials(q)
    if np >= nq
        succp = succprob(p)
        succq = succprob(q)
        res = np * kldivergence(Bernoulli{typeof(succp)}(succp), Bernoulli{typeof(succq)}(succq))
        return np == nq ? res : oftype(res, Inf)
```